docs/quickstart.md

```diff
@@ -0,0 +1,111 @@
+## Quickstart on a Single Host
+
+This tutorial assumes you are starting fresh and have no existing collectd agent running on your instance.
+
+#### Step 1: Download and install the agent
+
+```sh
+curl -sSL https://dl.signalfx.com/signalfx-agent.sh > /tmp/signalfx-agent.sh
+sudo sh /tmp/signalfx-agent.sh YOUR_SIGNALFX_API_TOKEN
+```
+
+Your SignalFx API Token can be obtained from the Organization->Access Token tab in [SignalFx](https://app.signalfx.com)
+
+More detailed installation steps to install via a config management tool or using a containerized agent can be found [here](../README.md#installation)
+
+#### Step 2: Configuration
+
+The default configuration file should be located at /etc/signalfx/agent.yaml
```

Comment:
It might be a good idea to put the file paths in back ticks `/etc/signalfx/agent.yaml`